### ISSUE TITLE
FEATURE REQUEST: SOCKS5/WEB Proxy settings

### ISSUE THREAD

**Issue by robco** [docker/kitematic#3184]
Many companies tend to limit network and require to use proxy to be set.

Please add SOCKS5/WEB proxy settings in order to be able to use Kitemanic on the limited corporate networks.

**Comment by muffl0n** [comment]
You can specify the proxy on the command line before starting Kitematic:
```
export http_proxy=http://user:pwd@myproxy:8080
export https_proxy=http://user:pwd@myproxy:8080
```

Unfortunately that only works halfway. With "--enable-logging" I'm getting this error message which tells me that proxy support in Kitematic seems to be only "half-done": 
```
[14614:1124/105640:WARNING:http_network_transaction.cc(365)] Blocked proxy response with status 407 to CONNECT request for kitematic.com:443.
```

**Comment by FrenchBen** [comment]
The proxy will also need to be setup within the actual VM as well. See our VPN section inside of the wiki.

**Comment by biomassives** [comment]
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  acmeideal@gmail.com
    Domain biomassiv.es has exceeded the max emails per hour (107/100 (107%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from o6.sgmail.github.com ([192.254.113.101]:26994)
	by chi-server32.websitehostserver.net with esmtps (TLSv1.2:ECDHE-RSA-AES128-GCM-SHA256:128)
	(Exim 4.89)
	(envelope-from <bounces+848413-b37a-greg=biomassiv.es@sgmail.github.com>)
	id 1eMLEH-00145Q-Id
	for greg@biomassiv.es; Tue, 05 Dec 2017 15:58:17 -0600
DKIM-Signature: v=1; a=rsa-sha1; c=relaxed/relaxed; d=github.com; 
	h=from:reply-to:to:cc:in-reply-to:references:subject:mime-version:content-type:content-transfer-encoding:list-id:list-archive:list-post:list-unsubscribe; 
	s=s20150108; bh=aDU3/nPiCLjubdVtrJUWm3MNNxc=; b=OhmDQd3xrTf/Zlzj
	6GcQI+IIOx8ksWj5or6HVLVMgdcKkuMLuGURJcIfnuuYkBjZoq87+xP6tLXJ9zYo
	AJNKAZR69ZkBBOB+8JbmHnuVDtpmxP/DfvCyyRPSrvEIaDe1Pzt0xtqGD4xg0PfT
	wd/6qMAP6/2MEzPmRxO5Ra4Cp3k=
Received: by filter0257p1las1.sendgrid.net with SMTP id filter0257p1las1-18192-5A271651-35
        2017-12-05 21:57:37.912892243 +0000 UTC
Received: from github-smtp2a-ext-cp1-prd.iad.github.net (github-smtp2a-ext-cp1-prd.iad.github.net [192.30.253.16])
	by ismtpd0007p1iad1.sendgrid.net (SG) with ESMTP id X8panWXlRL6jwJ3S-UBj3Q
	for <greg@biomassiv.es>; Tue, 05 Dec 2017 21:57:37.710 +0000 (UTC)
Date: Tue, 05 Dec 2017 21:57:37 +0000 (UTC)

**Comment by muffl0n** [comment]
@FrenchBen, that's surely correct. Unfortunately it does not help with this problem at all.
